.github/workflows/platformDeploy.yml

```diff
@@ -84,13 +84,15 @@ jobs:
         if: ${{ !fromJSON(env.SHOULD_DEPLOY_PRODUCTION) }}
         run: bundle exec fastlane android beta
         env:
+          RUBYOPT: '-rostruct' # TODO: Temporary fix until https://github.com/fastlane/fastlane/pull/21950 gets released
```

Comment:
Reminder: TODOs in the code are generally not a best practice - it's better to create a follow-up GitHub issue and link it to the code via a comment in the PR

Reply:
Fair point, resubmitted alongside a fix [here](https://github.com/Expensify/App/pull/45459).

Reply:
Fair point. I removed this code alongside another fix here and created an issue in place of the TODO